File: insert.php
<?php 
//echo phpinfo();
$mysql_host = "localhost";
$mysql_user = "root";
$mysql_password = "Wwyan107";
$mysql_database = "contact";


// $con = mysqli_connect($db_hostname,$db_username,$db_password, $db_database);
$con = mysqli_connect($mysql_host, $mysql_user, $mysql_password, $mysql_database);

if (!$con) {
		echo "Error:Unable to connect to MySQL." . PHP_EOL;
		echo "Debugging errno: " . mysqli_connect_errno() . PHP_EOL;
		echo "Debugging error: " . mysqli_connect_error() . PHP_EOL;
		exit;
}

//echo here;
//echo $con;

/*
mysqli_select_db($mysql_database, $con)
or die("Unable to select database: " . mysqli_error());
*/
    
if (isset($_POST['firstname']) &&
isset($_POST['lastname']) &&
isset($_POST['email']) &&
isset($_POST['message']))
{
   $FirstName = $_POST['firstname'];
   $LastName = $_POST['lastname'];
   $Email = $_POST['email'];
   $Message = $_POST['message'];

/*
echo $FirstName;
echo $LastName;
echo $Email;
echo $Message;
*/
		
  // $query = INSERT INTO info (FirstName, LastName, Email, Message) VALUES ('$FirstName', '$LastName', '$Email', '$Message');
  mysqli_query($con,"INSERT INTO info (firstname, lastname, email, message) 
  VALUES ('$FirstName', '$LastName', '$Email', '$Message')");



echo "Your request has been processed. Thank you for choosing Tiffany !";
		
/* if (!mysqli_query($query, $con)) 
      die ("INSERT failed: $mysql_query ". mysqli_error());
*/
} 


mysqli_close($con);

?>

File: insert.py
from flask import Flask, render_template, flash, redirect, url_for, session, request, logging
from flask_mysqldb import MySQL
from wtforms import Form, StringField, TextAreaField, PasswordField, validators

app = Flask(__name__)

app.config['MYSQL_HOST'] = 'localhost'
app.config['MYSQL_USER'] = 'root'
app.config['MYSQL_PASSWORD'] = 'Wwyan107'
app.config['MYSQL_DB'] = 'contact'
app.config['MYSQL_CURSORCLASS'] = 'DictCursor'
mysql = MySQL(app)


@app.route('/contact', methods=['GET', 'POST'])
def contact():
    if request.method == 'POST':
        firstname = request.form["firstname"]
        lastname = request.form["lastname"]
        email = request.form["email"]
        message = request.form["message"]
        cur = mysql.connection.cursor()

        result = cur.execute("INSERT INTO info (firstname, lastname, email, message) VALUES (%s, %s, %s, %s)", (firstname, lastname, email, message))
        mysql.connection.commit()

        cur.close()
    return render_template('contact.html')


if __name__ == '__main__':
    app.run(debug=True)
